.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+on:
+  push:
+    branches: [ master ]
```

Comment:
should this be main branch?

Reply:
Heh, well that will explain why this isn't running. That's what I get for copy and pasting this from one I wrote a long time ago :P